Automattic/wp-calypso
# Hosting Metrics: Add two scales on the first line chart

- Related to https://github.com/Automattic/dotcom-forge/issues/3274

## Proposed Changes

* Add two scales by defining a `scale` property inside a series.
* The name defined can be anything, in this example is `requests-per-min` but it could be `foo`.
* Not sure if it is possible to align the numbers in each line, and displaying lines per each scale is confusing. Maybe it's better to use different colors. In this PR I just hide the horizontal lines for the new scale.

## Testing Instructions



* Navigate to `/site-monitoring/${siteSlug}`
* Observe the two scales on the first chart


### Screencast

**Before**

https://github.com/Automattic/wp-calypso/assets/779993/c8de6c17-902b-4691-b6cf-f51285cf9db3 


**After**

https://github.com/Automattic/wp-calypso/assets/779993/48dba729-82f3-4b3a-a662-ec4510077d55



## Pre-merge Checklist
